packages/react-components/src/utils/parsing.tsx

```diff
@@ -146,3 +152,31 @@ export const parseTagNames = (poorText = false) =>
         // rehype-toc
         'nav',
       ];
+
+export function richTextToMarkdown(text = '', poorText = false): string {
+  let processor = unified().use(rehypeHtml, { fragment: true });
+  processor = processor.use(rehypeSanitize, {
+    tagNames: parseTagNames(poorText),
+    attributes: {
+      ...githubSanitizationSchema.attributes,
+      iframe: [
+        'src',
+        'title',
+        'width',
+        'height',
+        ['allow', 'fullscreen'],
+        'allowfullscreen',
+      ],
+    },
+  });
+
+  processor = processor.use(rehypeRemark).use(remarkStringify, {
+    bullet: '*',
+    fence: '~',
+    fences: true,
+    incrementListMarker: false,
+  });
```

Comment:
Is there a reason why you can not chain the `use` function calls?
```suggestion
  const processor = unified()
    .use(rehypeHtml, { fragment: true })
    .use(rehypeSanitize, {
    tagNames: parseTagNames(poorText),
    attributes: {
      ...githubSanitizationSchema.attributes,
      iframe: [
        'src',
        'title',
        'width',
        'height',
        ['allow', 'fullscreen'],
        'allowfullscreen',
      ],
    },
  })
  .use(rehypeRemark).use(remarkStringify, {
    bullet: '*',
    fence: '~',
    fences: true,
    incrementListMarker: false,
  });